contrib/objectstore/subscriber/secadvisor.go

```diff
@@ -36,11 +40,44 @@ type SecurityAdvisorFlow struct {
 	Start            int64
 	Last             int64
 	FinishType       flow.FlowFinishType
+	AName            string `json:"A_Name"`
+	BName            string `json:"B_Name"`
 }
 
 // SecurityAdvisorFlowTransformer is a custom transformer for flows
 type SecurityAdvisorFlowTransformer struct {
-	seenFlows *cache.Cache
+	seenFlows     *cache.Cache
+	gremlinClient *client.GremlinQueryHelper
+	containerName *cache.Cache
+}
+
+func (ft *SecurityAdvisorFlowTransformer) queryContainerName(ipString string) (string, error) {
+	node, err := ft.gremlinClient.GetNode(g.G.V().Has("Runc.Hosts.IP", ipString))
```

Comment:
It should be better to first find the interface having the NodeTID (meaning the capture node) from there find the related node with the IP. I'm saying that because just checking with the IP you may have conflict.